lib/fuzzy-finder-view.coffee

```diff
@@ -24,6 +24,8 @@ class FuzzyFinderView extends SelectListView
         @splitOpenPath (pane, item) -> pane.splitDown(items: [item])
       'pane:split-up': =>
         @splitOpenPath (pane, item) -> pane.splitUp(items: [item])
+      'fuzzy-finder:alternate-confirm': =>
```

Comment:
Changed to `invert-confirm`, but I agree that it could be made clearer still. How important is brevity? Could be something like `invert-pane-search-preference-confirm`?